tests/profiler/user_workflow_fixtures.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+
+# TODO: Move these fixtures to integration tests
+
+@pytest.fixture(scope="module")
+def alice_columnar_table_single_batch():
+    profiler_config = """
+variables:
+  max_user_id: 999999999999
+  min_timestamp: 2004-10-19 10:23:54
+rules:
+  my_rule_for_user_ids:
```

Comment:
Does the `Rule` need a `class_name` and the optional `module_name` in order to be instantiated?  Any other configuration for instantiation?

Reply:
Yes - this configuration is more of an example of what the user will type in and in this case we will assume defaults for the `class_name` and `module_name`. In our design doc there is a more comprehensive version of this configuration with all optional / assumed items shown. Do you think it would also be helpful to have that version of the configuration in this fixture so we can test both produce the same expectation suite?

Reply:
Actually for `Rule` it does not need `class_name` and `module_name` in the yaml config. We instantiate `Rule` objects in the constructor of `Profiler` so we don't currently allow custom `Rule` objects. I think this is fine (at least for now) since all of the intended configurability is in the DomainBuilders, ParameterBuilders & ExpectationConfigurationBuilders.

Reply:
@anthonyburdi My understanding (could be wrong) is that for as long as an entity in our system has *any* representation in the `YAML` configuration, we should be also including the instantiation information (e.g., `class_name`, etc.) in its respective section of the `YAML` configuration.  Hence, we would then not be instantiating `Rule` directly in our code, but doing it `wholesale` (i.e., together with all of its parameters) using `instantiate_class_from_config()`).  Happy to discuss to make sure we are taking the most fitting approach. /cc @talagluck

Reply:
That makes sense @alexsherstinsky. In general, in the proposed `simplified_profiler_config` configuration the goal is brevity and removing instantiation information when we can assume a sensible default for simple configurations. However I don't think we have considered extending the `Rule` class and though I can't think of a good example use case right now that is not to say one doesn't exist. This would also require a change to the instantiation of the `Rule` objects in `Profiler.__init__()`.
I added the following open question to our design doc, perhaps we should discuss in an upcoming design review? 
```
Should the Rule class be extendable (i.e. use a class_name and module_name to instantiate if included in the Profiler configuration yaml)?
```